src/decouplededitoruiview.js

```diff
@@ -68,13 +73,6 @@ export default class DecoupledEditorUIView extends EditorUIView {
 	render() {
 		super.render();
 
-		this.registerChild( [ this.toolbar, this.editable ] );
-	}
-
-	/**
-	 * @inheritDoc
-	 */
-	get editableElement() {
```

Comment:
As above, `editableElement` getter is part of the editor intercase which can be used by plugins. I really hope we will close https://github.com/ckeditor/ckeditor5/issues/1445 soon and remove it, but at this moment we should keep it.